### ISSUE TITLE
Why does the dataset to be predicted need celltype columns

### ISSUE THREAD

**Issue by Liripo** [TencentAILabHealthcare/scBERT#7]
https://github.com/TencentAILabHealthcare/scBERT/blob/a6998d3eaadb0cddab6efd777d5138f644c5754d/predict.py#L74, Why does the dataset to be predicted need celltype columns？The tutorials are not very clear.

**Comment by TencentAILabHealthcare** [comment]
The cell types to be predicted are based on the reference dataset which provides the cell type labels that you are interested in and is used for fine-tuning before prediction.

**Comment by Liripo** [comment]
@TencentAILabHealthcare https://github.com/TencentAILabHealthcare/scBERT/blob/55e1b46aa614d0f382541d204c7f5e4ec1927169/finetune.py#L123,
Wasn't it fine-tuned in the previous script finetune.py?

**Comment by TencentAILabHealthcare** [comment]
Yes, the finetune is based on reference dataset, and then the fine-tuned model is used for prediction. As the comment says, you can store the label dict in the fine-tune step.

**Comment by Liripo** [comment]
@TencentAILabHealthcare 
But isn't predict.py the input to the dataset to be predicted, which shouldn't have cell type information in the beginning?

**Comment by TencentAILabHealthcare** [comment]
Have updated the scripts and the tutorial. The cell type information is saved in the label and label dict files during fine-tuning, and then the information is loaded for prediction.
